packages/wonder-blocks-link/src/components/link-core.js

```diff
@@ -92,8 +96,14 @@ const sharedStyles = StyleSheet.create({
     },
 });
 
-const _generateStyles = (kind, light, visitable) => {
-    const buttonType = kind + light.toString() + visitable.toString();
+const _generateStyles = (inline, kind, light, visitable) => {
+    // Even though `inline` is defined and set the same as all the other
+    // boolean props, it is coming up as `undefined` in tests, which means
+    // the toString() function can't be used. Setting it explicitly to avoid
+    // a TypeError.
```

Comment:
I have NO idea why this is happening. 😕 Maybe someone can help me look at this before I land this PR?

Reply:
what value `buttonType` is using? I'm not sure what was the original intention, but to me it looks like it's setting something like: `primarytruetrue`? I'm not sure if that's what we want (to clarify this is not something that you introduced).

Reply:
Yeah, I believe that is right. Each variant of button has its own style saved in this `styles` array with `buttonType` being the index. I'm not sure why it's done this way.

Reply:
I wonder what would happen if you move `inline` after `kind`? maybe it would also help to separate each condition:

e.g.
```
const buttonType = `${kind}-${inline}-${light}-${visitable}`;
```